enclave/core/debugmalloc.c

```diff
@@ -618,6 +636,27 @@ int oe_posix_memalign(void** memptr, size_t alignment, size_t size)
     return rc;
 }
 
+void* oe_aligned_alloc(size_t alignment, size_t size)
+{
+    void* p = NULL;
+    if (oe_use_debug_malloc)
```

Comment:
We need to validate alignment and size according to spec.

Reply:
Hi Anand, I added the param check as per [https://linux.die.net/man/3/aligned_alloc](url). Can you check again?